samza-sql/src/main/java/org/apache/samza/sql/data/SamzaSqlExecutionContext.java

```diff
@@ -38,7 +40,7 @@
    * The variables that are shared among all cloned instance of {@link SamzaSqlExecutionContext}
    */
   private final SamzaSqlApplicationConfig sqlConfig;
-  private final Map<String, UdfMetadata> udfMetadata;
+  private final Map<String, List<UdfMetadata>> udfMetadata;
```

Comment:
So here, `UdfMetadata`  is for `SamzaSQLUDFMethod` ,  and `Map<String, List<UdfMetadata>>` is for `SamzaSqlUdf`?